Bottom: ed3644ffbfbff18449d2c0d05ed7784881345514
Top:    a4e56302c17f5ed0a64ff9c5efcd38e25bc3da26
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-22 20:32:25 +0000

Refresh of attempt-to-split-out-clip_box

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3f12a20..b4be3a5 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1717,7 +1717,7 @@ fill_board_outline_holes (hidGC gc)
 
   p = polygon;
   do {
-    PolygonHoles (&p, gc->hid_draw_>clip_box, fill_outline_hole_cb, &info);
+    PolygonHoles (&p, gc->hid_draw->clip_box, fill_outline_hole_cb, &info);
   } while ((p.Clipped = p.Clipped->f) != polygon.Clipped);
 
 //  poly_FreeContours (&polygon.NoHoles);
